modules/utils/build.rs

```diff
@@ -7,50 +70,72 @@ fn main() {
             println!("cargo:rustc-cfg=onnx_runtime_env_var_set");
         },
         Err(_) => {
-            #[cfg(not(windows))]
-            {
-                let _ = Command::new("sh")
-                .arg("-c")
-                .arg("cargo new onnx_driver && cd onnx_driver && echo 'ort = \"1.16.2\"' >> Cargo.toml
-                ")
-                .status()
-                .expect("failed to execute process");
-            }
-
-            #[cfg(windows)]
-            {
-                // let _ = Command::new("cmd")
-                // .args(&["/C", "cargo new onnx_driver && cd onnx_driver && echo ort = \"1.16.2\" >> Cargo.toml"])
-                // .status()
-                // .expect("failed to execute process");
-                let _ = Command::new("powershell")
-                    .arg("-Command")
-                    .arg("cargo new onnx_driver; Set-Location onnx_driver; Add-Content -Path .\\Cargo.toml -Value 'ort = \"1.16.2\"'")
-                    .status()
-                    .expect("failed to execute process");
-            }
-
-            #[cfg(not(windows))]
-            {
-                let _ = Command::new("sh")
-                    .arg("-c")
-                    .arg("cd onnx_driver && cargo build")
-                    .status()
-                    .expect("failed to execute process");
-            }
-
-            #[cfg(windows)]
-            {
-                let _ = Command::new("cmd")
-                    .args(&["/C", "cd onnx_driver && cargo build"])
-                    .status()
-                    .expect("failed to execute process");
-            }
+            let target = std::env::var("TARGET").unwrap();
+
+            let (prebuilt_url, prebuilt_hash) = match target.as_str() {
+                "aarch64-apple-darwin" => (
+                    "https://parcel.pyke.io/v2/delivery/ortrs/packages/msort-binary/1.16.3/ortrs-msort_static-v1.16.3-aarch64-apple-darwin.tgz",
```

Comment:
The versions and shared prefixes can be extracted. This guarantees that we don't have version typos then.

Reply:
I've now moved over to just using the download binaries feature in the `ort` for the `dev-dependencies` and then using the binary for the include bytes, all builds are now passing